common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1,4 +1,4 @@
-(function (define) {
+(function (define, Draggabilly) {
```

Comment:
@singingwolfboy ah cool, thanks!

@OmarIthawi can you investigate the validity of the Javascript in the file? Maybe try [running the acceptance test locally](https://github.com/edx/edx-platform/blob/master/docs/en_us/internal/testing.md#running-lettuce-acceptance-tests), until at minimum the JS loading error is resolved. The local command would be
```
paver test_acceptance -s cms --extra_args="cms/djangoapps/contentstore/features/component.feature -s 4"
```